File: app/src/main/java/com/sample/app/api/customer/CustomerAPI.java
package com.sample.app.api.customer;

import com.sample.app.entity.Customer;
import com.google.gson.JsonElement;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;

public interface CustomerAPI {
    @GET("js/customers_mysql.php")
    Call<JsonElement> getCustomers1();

    @GET("js/customers_mysql.php")
    Call<List<Customer>> getCustomers2();
}

File: app/src/main/java/com/sample/app/api/customer/CustomerAPI.kt
package com.sample.app.api.customer

import com.google.gson.JsonElement
import com.sample.app.entity.Customer
import retrofit2.Call
import retrofit2.http.GET

interface CustomerAPI {
    @get:GET("js/customers_mysql.php")
    val customers1: Call<JsonElement>

    @get:GET("js/customers_mysql.php")
    val customers2: Call<List<Customer>>
}
